app/api.js

```diff
@@ -41,6 +45,23 @@ function header() {
   }
 }
 
+function authenticatedFetch(url, args = {}) {
+  return fetch(url, {
+    ...args,
+    ...credentials(),
+  }).then(checkStatus)
+    .then(parseJSON)
+    .catch(err => err.json())
+    .then(res => {
+      const errors = res.errors
+      if (errors && errors.map(e => e.message).find(m => m === UNAUTHORIZED)) {
```

Comment:
Could we use status 403 to check this instead of relying on the backend sending this exact error message?
Since we have the `req` in the graphQL resolver I think it should be possible.

Reply:
we would need `res` instead of `req` right?

Reply:
Yep, we could just pass it in?
https://github.com/studieresan/overlord/blob/master/src/server.ts#L154

Reply:
cool, will try